src/main/java/core/basesyntax/Mail.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.service.Inventory;
+import core.basesyntax.service.InventoryFromCsv;
+import core.basesyntax.service.Report;
+import core.basesyntax.service.ReportToCsv;
+import core.basesyntax.storage.Storage;
+
+public class Mail {
+    public static void main(String[] args) {
+        Storage storage = new Storage();
+
+        Inventory inventory = new InventoryFromCsv();
+        inventory.synchronizeWithTheStorage();
+
+        Report report = new ReportToCsv();

```

Comment:
Ensure that services are independent and called in the correct order. The checklist suggests that each service should return specific data types and pass them to the next service, maintaining independence and adhering to SOLID principles .